osu.Game.Rulesets.Taiko/Mods/TaikoModJudgementChallenge.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using osu.Game.Rulesets.Mods;
+using osu.Framework.Bindables;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Overlays.Settings;
+
+
+namespace osu.Game.Rulesets.Taiko.Mods
+{
+    public class TaikoModJudgementChallenge : ModJudgementChallenge
+    {
+        [SettingSource("Maximum misses", "Maximum number of misses before fail.", SettingControlType = typeof(SettingsSlider<int, JudgementMaxSlider>))]
+        public BindableNumber<int> MaxMiss { get; } = new BindableInt
+        {
+            Default = 26,
+            Value = 26,
+            MinValue = 0,
+            MaxValue = 26
+        };
```

Comment:
I viscerally disagree with this on a conceptual level. I argue that enforcing judgement counts like this shouldn't be a thing and is too much customisation.

Reply:
Also on this point: note that with this degree of customisation you can set completely nonsensical bounds here, e.g. you can have `MaxMiss = 10` and `MaxMeh = 5`, which to me makes zero sense.